app/models/external_api_client.rb

```diff
@@ -41,7 +47,11 @@ class ExternalApiClient < ApplicationRecord
   validates :drive_every, presence: true, if: -> { drive_strategy == ExternalApiClient::DRIVE_STRATEGIES[:cron] }
 
   validates :drive_every, inclusion: { in: ExternalApiClient::DRIVE_INTERVALS.keys.map(&:to_s) }, allow_blank: true, allow_nil: true
-  validates :model_definition, safe_executable: true
+  validates :model_definition, safe_executable: { skip_keywords: ['render'] }
```

Comment:
@Pralish could you make the skippable keywords a const (also, how is `render` a blocked world)